File: Itinerario 12 - Controls + Reports + Charts/Chart2/UI_C/frmXml.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Windows.Forms.DataVisualization.Charting;

namespace UI_C
{
    public partial class frmXml : Form
    {
        public frmXml()
        {
            InitializeComponent();
        }

  
 
     

        private void frmCSV_Load(object sender, EventArgs e)
        {
            CargarDatosXML();
        }


        void CargarDatosXML()
        {   
            // Inicializo una instancia de un Dataset
            DataSet DS = new DataSet();
            // Leeo el achivo XML y lo almaceno en el Dataset.
            DS.ReadXml("Personas.xml");
            chart1.DataSource = DS;
            // Especifique los miembros de datos de argumento y valor para la serie.
            chart1.Series[0].XValueMember = "nombre";
            chart1.Series[0].YValueMembers= "saldo";
            //digo q tipo de grafico quiero 
            chart1.Series[0].ChartType = SeriesChartType.StackedColumn;
            //y si lo quiero en 3D
            chart1.ChartAreas[0].Area3DStyle.Enable3D = true;
            chart1.DataBind();
        }

   
    }
}

File: (FINAL)/(XML)/Chart/UI_VB/frmXML.vb
Imports System.Windows.Forms.DataVisualization.Charting

Public Class frmXML



    Private Sub frmXML_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        CargarDatosXML()
    End Sub


    Sub CargarDatosXML()
        'Inicializo una instancia de un Dataset
        Dim DS As New DataSet()
        ' Leeo el achivo XML y lo almaceno en el Dataset.
        DS.ReadXml("Personas.xml")
        Chart1.DataSource = DS
        ' Especifique los miembros de datos de argumento y valor para la serie.
        Chart1.Series(0).XValueMember = "nombre"
        Chart1.Series(0).YValueMembers = "saldo"
        'digo q tipo de grafico quiero 
        Chart1.Series(0).ChartType = SeriesChartType.StackedColumn
        'y si lo quiero en 3D
        Chart1.ChartAreas(0).Area3DStyle.Enable3D = True
        Chart1.DataBind()
    End Sub
End Class